src/patterns/components/spinners/dark.hbs

```diff
@@ -0,0 +1,8 @@
+---
+name: Dark Spinner
+notes: Dark variant is used when its needed to put a spinner against a light background.
```

Comment:
For the notes: sentence here, I thought something like this might improve the sentence.

> The Dark Spinner variant is used to improve visibility when the spinner is placed against a light background color.